src/socket/dhcpv4.rs

```diff
@@ -1358,6 +1371,39 @@ mod test {
         }
     }
 
+    #[rstest]
+    #[case::ip(Medium::Ethernet)]
+    #[cfg(feature = "medium-ethernet")]
+    fn test_min_max_renew_timeout(#[case] medium: Medium) {
+        let mut s = socket_bound(medium);
+        // Set a minimum of 45s and a maximum of 120s
+        let config = RetryConfig {
+            max_renew_timeout: Duration::from_secs(120),
+            min_renew_timeout: Duration::from_secs(45),
+            ..s.get_retry_config()
```

Comment:
Since Default is implemented, could we not use that to complete the struct initialization? Then we don't need to add the `get_retry_config` function, unless you think that function is helpful to keep it.

Reply:
For this test code, you're right that `Default::default()` would work equally well to `get_retry_config`.

For my use-case, it is convenient to be able to set some parameters of the config while leaving others unchanged, which the `get_retry_config` method enables.